### PR TITLE
Event list raw - done

### PR DESCRIPTION
**Pull request by rifqiizz**
Mas @jordanmaulana @indrazm 

Tambahan raw Event list. Utk pengerjaan UI di task selanjutnya.

Cek di url localhost:3000/Events

### CODE REVIEW

File: src/components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        headers: {
+              'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({"_id":`${id}`,"name":`${name}`,"tanggal":`${tanggal}`,"jenis":`${jenis}`,"keterangan":currentDesc})
+      //body: JSON.stringify({ content: currentContent }),
+    });
+    const data = await res.json();
+    setOnEdit(false); */
+    router.refresh();
+  }
```
Comment (jordanmaulana) [inline src/components/Card.jsx:48]: ini dihapus aja mas, karena targetnya nanti memang ga ada delete & update

**Review by jordanmaulana — COMMENTED** [review]
Kalo ada waktu,
penulisan code nya disamakan aja seperti contoh projeknya mas Indra. Zip nya sudah ada di channel #announcement Discord.

File: src/components/Card.jsx (hunk 1 above)
Comment (rifqiizz) [reply src/components/Card.jsx:48]: Siap

File: src/app/Events/page.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { Card } from "@/components/Card";
+
+async function getEvents(){
+  const res = await fetch("https://eventmakers-api.vercel.app/api/event",{
```
Comment (indrazm) [inline src/app/Events/page.js:4]: Move this to config file or folder

**Comment by rifqiizz** [comment]
> Kalo ada waktu, penulisan code nya disamakan aja seperti contoh projeknya mas Indra. Zip nya sudah ada di channel #announcement Discord.

Ini yg bagian mana? Waktu kirim data kah?

File: src/app/Events/page.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    //cache: "no-store",  
+    cache: "force-cache",
```
Comment (indrazm) [inline src/app/Events/page.js:6]: not `force-cache` but `no-store`

**Review by indrazm — COMMENTED** [review]
I have couple of comments guys. Please fix.

File: src/app/Events/page.js (hunk 3 above)
Comment (rifqiizz) [reply src/app/Events/page.js]: Done

File: src/app/Events/page.js (hunk 2 above)
Comment (rifqiizz) [reply src/app/Events/page.js:4]: Mas @indrazm @jordanmaulana ini sy perlu cek video lg, sekilas compare folder bedanya di sini ya utk kelola API urlnya:

- config/apiUrl.js
- .env

Please confirm?

Comment (jordanmaulana) [reply src/app/Events/page.js:4]: Yess. Please pull main branch and you'll know it ;)
Also yeah please checkout the video wkkwkwkwk.
We overslept for the last 1 hour. I have seen the video and did what I did to match it.

**Review by indrazm — APPROVED** [review]
lgtm.

**Pull request merged**